tests/subscription/make-payment-failure.spec.js

```diff
@@ -0,0 +1,14 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test('Make Payment (failure)', async ({ page }) => {
+    // call to invalid token
+    await page.goto('/admin/makepayment/12345');
+
+    // check error message
+    await expect(page.locator('text="Payment failed for RecurringDetailReference"')).toBeVisible();
+    // check error icon
+    await expect(page.locator('.status-image >> visible=true')).toBeVisible();
+
+    await expect(page.locator('text="Return"')).toBeVisible();
```

Comment:
I know we do the same for other tests, but just wanted to challenge you with this question: Should we include this button in this particular test (we're testing the payment functionality here, not necessarily if we can return)?

Reply:
This only checks the Return link is visible